pages/index.html

```diff
@@ -4,9 +4,49 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Mexican Cousine</title>
+    <link href="styles/style.css" rel="stylesheet">
+    <link href="styles/style-index.css" rel="stylesheet">
 </head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+<body class="container">
+  <header>
+    <nav class="nav-bar">
+      <a href="facts.html">Fun Facts!</a>
+      <a href="gallery.html">Gallery!</a>
+    </nav>
+  </header>
+    <h1 class="shimmer">Mexican Cuisine: A Heritage of Tradition</h1>
```

Comment:
Everything after the `header` looks like it's basically an `article`. You could use that!